src/ds/nonstd.cpp

```diff
@@ -14,21 +12,6 @@
 
 namespace ccf::nonstd
 {
-  // Implementations for ccf/ds/nonstd.h
-  void to_upper(std::string& s)
-  {
-    std::transform(s.begin(), s.end(), s.begin(), [](unsigned char c) {
-      return std::toupper(c);
-    });
-  }
-
-  void to_lower(std::string& s)
-  {
-    std::transform(s.begin(), s.end(), s.begin(), [](unsigned char c) {
-      return std::tolower(c);
-    });
-  }
-
   // Implementations for ds/nonstd.h
   std::string camel_case(
```

Comment:
I suggest moving this to endpoints/something.cpp, either endpoint_registry.cpp or its own file, but I don't think it should be under nonstd:: or under nonstd/.

Reply:
I don't think it's a good place to place this either, but TBH it's been already too much jerking around this, so let's bury it under `endpoints::` in `endpoint_registry.coo` and move on.

Reply:
Done